index.md

```diff
@@ -8,6 +8,356 @@ This is some welcome text.
 
 This is some helpful description of the C0111 error.
 
+
+### R0204: Redefined variable type {#R0204}
+
+This error occurs when the type of a variable changes inside a method or a 
+function. See the examples below.
+
+```python
+def obvious_variable_redefinition(arg):
+    """Redefinition of <variable-name> type from <type_1> to <type_2>"""
+    arg = 1
+    arg = "string"  # redefined variable type
+
+def other_redefinition_example(arg):
+    """Redefinition of <variable-name> type from <type_1> to <type_2>"""
+    if arg == 3:
+        var = "8"
+    else:
+        var = 8  # redefined variable type
+```
+
+
+### E0401: Import error {#E0401}
+
+The module is unable to be imported. Check the spelling of the module name, or
+whether the module is on disc.
```

Comment:
Don't say "on disc." You can say "is installed", or "is in the current directory", or something similar.